tests/feature/test_amino_acid_f.py

```diff
@@ -32,5 +31,5 @@ def test_add_features():
 
     for node in graph.nodes:
         if node.id == variant.residue:  # GLY -> SER
-            node.features[FEATURENAME_SIZEDIFFERENCE] > 0
-            node.features[FEATURENAME_HYDROGENBONDDONORSDIFFERENCE] > 0
+            node.features[FEATURENAME_SIZEDIFFERENCE] > 0 # pylint: disable=pointless-statement

```

Comment:
Is this line supposed to do anything?

Reply:
I think that should be preceded by assert. I must have forgotten.

Reply:
perfect I'll add it before merging